tests/unittests/Foundation/NSJSONSerializationTests.m

```diff
@@ -124,13 +124,13 @@ void VerifyDataWithJSONObjectThrows(id testObject) {
     id testObject6 = @{ @"foo" : @"1", @"bar" : @"2" };
     id testObject7 = @{ @"foo" : @{ @"bar" : @"1" }, @"foo2" : @{ @"bar2" : @"2" } };
     id testObject8 = [NSUUID UUID];
-    id testObject9 = @{ @"key" : @"value", @"nullkey" : [NSNull null], @"array" : @[ @{ @"subnullkey" : [NSNull null] } ] };
```

Comment:
It's determined by the order in which CFDictionary returns the keys for fastenumeration, which I a) really don't want to mess with and b) think is far outside of the scope of this issue.

---
In reply to: [80824620](https://github.com/Microsoft/WinObjC/pull/1043#discussion_r80824620) [](ancestors = 80824620)